docs/onboarding/Onboarding.md

```diff
@@ -0,0 +1,92 @@
+## Onboarding guide
+
+Playbook dispatcher can help you a great deal when you have to keep track of multiple playbook runs across multiple hosts. When you dispatch playbooks using Playbook Dispatcher, you do not need to worry about constructing cloud connector requests or keeping track of playbook run statuses. Playbook Dispatcher will manage all these things for you.
```

Comment:
As this is intended for a technical audience IMO a less conversational style would be appropriate, e.g. "Playbook Dispatcher is a service that simplifies running Ansible Playbooks on connected hosts. This guide explains how an application be integrated with it in order to benefit from:
* common API for dispatching playbooks
* automated tracking of playbook state
* common model for representing playbook results
* built-in authorization support"